src/tests/conformance/mod.rs

```diff
@@ -12,6 +12,14 @@ mod query;
 mod stateful;
 mod status;
 
+pub const PUBLIC_KEY_TYPES: &[u8] = &[
```

Comment:
haven't we added this one elsewhere recently :detective:?

Reply:
ah, it's just moved :+1:

Reply:
Yes, but now it has better name and it's moved up to be accessible in more places.